table/block_based_table_reader.h

```diff
@@ -609,6 +609,12 @@ class BlockBasedTableIterator : public InternalIteratorBase<TValue> {
     assert(Valid());
     return block_iter_.key();
   }
+  Slice user_key() const override {
+    if (key_includes_seq_) {
```

Comment:
Would not it be cleaner to return block_iter_.user_key()?

Reply:
oops. done.